scaffolder/publish-standalone-tech-docs/publish-standalone-tech-docs.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: publish-standalone-tech-docs
+  title: Publish a new standalone TechDocs repository and register with Roadie Backstage
+  description: "Publish a new repository with a catalog-info.yaml file with your company's standalone TechDocs details."
+spec:
+  owner: unspecified # TODO: Define who the owner of this template is. This is likely the team who own the relationship with Roadie. e.g. group:customer-name/platform
+  type: service
+
+  # These are the steps which are rendered in the frontend with the form input.
+  parameters:
+    - title: Repository Details # Title of the first step.
+      required:
+        - repoName
+      properties:
+        repoHost:
+          type: string
+          default: github.com
+          ui:widget: hidden
+
+        gitHubOrg:
+          type: string
+          default: roadie-demo # TODO: Set your GitHub Org value here.
+          ui:widget: hidden
+
+        repoName:
+          title: Repository Name
+          type: string
+          default: company-docs # TODO: Set your default name for the documentation repository here.
+
+        defaultBranch:
+          title: Repository Branch
+          type: string
+          default: main # TODO: Define your expected default branch.
+          enum: ['main']
```

Comment:
enum with 1 item?? surely you can leave this out?

Reply:
The expectations is they would add their list of expected values but I guess they would probably normalize to a single default (not intermix main and master).

Reply:
I'd just cut this. It's rare that someone is going to want to change the name of the default branch.

Reply:
less is more.